core/app/models/spree/promotion.rb

```diff
@@ -40,6 +40,7 @@ class Promotion < Spree::Base
         where(table[:expires_at].eq(nil).or(table[:expires_at].gt(time)))
     end
     scope :applied, -> { joins(:order_promotions).uniq }
+    scope :applied_automatically, -> { where(apply_automatically: true) }
```

Comment:
I do see that `applied_automatically` matches the naming pattern for `advertised`, it just feels a bit confusing given the `applied` scope.